sites/public/src/components/shared/MapControl.tsx

```diff
@@ -0,0 +1,87 @@
+import { GoogleMap } from "@react-google-maps/api"
+
+// import { Icon, IconFillColors, ViewItem, t } from "@bloom-housing/ui-components"
+import { Icon } from "@bloom-housing/doorway-ui-components"
+
+const mapControlBorderRadius = 100
+const mapControlBorderColor = "#5c7088"
+
+export interface MapControlProps {
+  mapRef: RefObject<GoogleMap>
+}
+
+const controlStyle = {
+  padding: "10% 0",
+  backgroundColor: "white",
+  borderColor: `${mapControlBorderColor}`,
+  borderStyle: "solid solid solid solid",
```

Comment:
(note that there are some SCSS stylistic nits e.g. no hardcoded colors, unnecessary styles -- will save for after the scss moving)